### PR TITLE
Feature/add run extension

### PR DESCRIPTION
**Pull request by TSteele27**

### CODE REVIEW

File: packages/bundle/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# @b-mo/bundle
+
+This packages loads and resolves a dependency bundle. It allows you to easily extend dependency bundles
```
Comment (BReul) [inline packages/bundle/README.md]:
```suggestion
This module packages, loads, and resolves a dependency bundle. It allows you to easily extend dependency bundles
```

File: packages/bundle/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+so that you can compose applications together easily.
+
+Install the bundle package using whatever package manager you are using these days:
+
+```npm i @b-mo/bundle```
+
+```yarn add @b-mo/bundle```
+
+
+By default then bundle will be loaded from the current working directory but you can override it with params
```
Comment (BReul) [inline packages/bundle/README.md]:
```suggestion
By default, bundle will be loaded from the current working directory, but you can override it with params
```

File: packages/cli/CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,8 @@
 # CHANGELOG
 
+v0.6.0
+- [CHANGE] Use Bundle and new composable module pattern
+- [CHANGE] Refactor portions of the code to be bmo modules instead of
```
Comment (BReul) [inline packages/cli/CHANGELOG.md]: this sentence isn't finished

**Review by BReul — DISMISSED** [review]

**Review by rajeshsusai — DISMISSED** [review]

File: packages/cli/CHANGELOG.md (hunk 3 above)
Comment (TSteele27) [inline packages/cli/CHANGELOG.md]:
```suggestion
- [CHANGE] Refactor portions of the code to be bmo modules.
```

Comment (TSteele27) [reply packages/cli/CHANGELOG.md]: changed!

**Review by rajeshsusai — APPROVED** [review]

**Review by srpiatt — APPROVED** [review]

**Review by cteung — APPROVED** [review]

**Pull request merged**